packages/squiggle-lang/src/rescript/Reducer/Reducer.resi

```diff
@@ -7,7 +7,20 @@ module MathJs = Reducer_MathJs
 
 @genType
 type expressionValue = ReducerInterface_ExpressionValue.expressionValue
-
+@genType
+type externalBindings = ReducerInterface_ExpressionValue.externalBindings
 @genType
 let evaluate: string => result<expressionValue, Reducer_ErrorValue.errorValue>
+@genType
+let evaluateWBindings: (
+  string,
+  externalBindings,
+) => result<expressionValue, Reducer_ErrorValue.errorValue>
+@genType
+let evaluatePartialWBindings: (
```

Comment:
maybe ``evaluateAndReturnPartial`` or ``evaluateUsingExternalBindingsAndReturningPartial``?

Reply:
renamed to `evaluatePartitalUsingExternalBindings`
Yes we are evaluating a partial code
And we are using external bindings